benchmarks/benchmark.py

```diff
@@ -0,0 +1,68 @@
+'''
```

Comment:
Make this be a description of what the benchmark does and some example usage. See cvt_add.py https://github.com/icaros-usc/pyribs/blob/master/benchmarks/cvt_add.py for an example